akamai/cache-client.go

```diff
@@ -0,0 +1,236 @@
+package akamai
+
+import (
+	"bytes"
+	"crypto/hmac"
+	"crypto/rand"
+	"crypto/sha256"
+	"encoding/base64"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"strings"
+	"time"
+
+	"github.com/letsencrypt/boulder/Godeps/_workspace/src/github.com/cactus/go-statsd-client/statsd"
+	"github.com/letsencrypt/boulder/Godeps/_workspace/src/github.com/jmhodges/clock"
+	"github.com/letsencrypt/boulder/core"
+
+	blog "github.com/letsencrypt/boulder/log"
+)
+
+const (
+	purgePath       = "/ccu/v2/queues/default"
+	timestampFormat = "20060102T15:04:05-0700"
```

Comment:
@rolandshoemaker explained this to me offline and now I get it. The 0700 is actually a placeholder and Go will insert the machine's time zone offset there.